packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -470,6 +474,27 @@ static Object clusterToJson(String clusterManagerId, Cluster<MarkerBuilder> clus
     return data;
   }
 
+  static Messages.PlatformCluster clusterToPigeon(
```

Comment:
The reason these methods are mostly new instead of edits (e.g., this is a modified copy of `clusterToJson` which still exists just above) is that the JSON methods are still used by the parts of the API that haven't been converted to Pigeon yet. Over time, they will be eliminated as more APIs are moved to Pigeon.